Sources/Dictionary/dictionaryManager.swift

```diff
@@ -0,0 +1,19 @@
+import OS
+
+public class DictionaryManager {
+	public let gameWords = words
```

Comment:
I am going to try a new approach here. I am going to inject words into dictionaryManager as a class property. Correct me if I am approaching that wrong.